test/IntegrationTests/TicketService/test_create_custom_fields.py

```diff
@@ -0,0 +1,160 @@
+import unittest
+import requests
+import logging
+import docker
+# Configure logging
+logging.basicConfig(
+    level=logging.INFO,
+    format='%(asctime)s - %(levelname)s - %(message)s'
+)
+logger = logging.getLogger(__name__)
+
+class TestCustomFieldAPI(unittest.TestCase):
+    BASE_URL = "http://localhost:8000"
+    MYSQL_CONTAINER_NAME = "mysql_db"
+
+    def setUp(self):
+        """Setup function to initialize valid customer GUID."""
+        logger.info("=== Initializing test setup ===")
```

Comment:
You dont need these lines

Remove it

        """Setup function to initialize valid customer GUID."""
        logger.info("=== Initializing test setup ===")
        self.docker_client = docker.from_env()

        # Ensure MySQL container is running
        mysql_container = self.docker_client.containers.get(self.MYSQL_CONTAINER_NAME)
        if mysql_container.status != "running":
            logger.info("Starting MySQL container for tests.")
            mysql_container.start()

Reply:
Removed sir